components/service/pocket/src/main/java/mozilla/components/service/pocket/Arguments.kt

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.pocket
+
+/**
+ * A collection of shared argument validation functions.
+ */
+internal object Arguments {
+
+    fun assertIsValidUserAgent(userAgent: String) {
+        assertIsNotBlank(userAgent, "user agent")
+    }
+
+    fun assertIsNotBlank(input: String, exceptionIdentifier: String) {
+        if (input.isBlank()) throw IllegalArgumentException("Expected non-blank $exceptionIdentifier")
+    }
```

Comment:
nit: Maybe Kotlin's [require](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/require.html) is an option here too?

Reply:
I didn't know this was a thing and it changes my life – thank you. :P